examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -614,9 +607,7 @@ class _AnimationDemoHomeState extends State<AnimationDemoHome> {
                 child: IconButton(
                   icon: const BackButtonIcon(),
                   tooltip: 'Back',
-                  onPressed: () {
-                    _handleBackButton(appBarMidScrollOffset);
-                  },
+                  onPressed: () { _handleBackButton(appBarMidScrollOffset); },
```

Comment:
on this i could go either way

Reply:
The problem is that the formatter need a rule to decide to apply one line block or expanded block.